src/pages/_app.tsx

```diff
@@ -0,0 +1,57 @@
+import { Grid } from "@chakra-ui/react";
+import type { AppProps } from "next/app";
+import Head from "next/head";
+import type { FunctionComponent } from "react";
+import "index.css";
+import { DevPreviewBanner } from "components/DevPreviewBanner";
+import { Footer } from "components/Footer";
+import { Header } from "components/Header";
+import { ExternalLinkWarningProvider } from "contexts/ExternalLinkWarning";
+import { ShortbreadProvider } from "contexts/Shortbread";
+import { Theme } from "contexts/Theme";
+
+const App: FunctionComponent<AppProps> = ({ Component, pageProps }) => {
+  return (
+    <ShortbreadProvider>
+      <Head>
```

Comment:
Also since you are duplicating the <Head> calls, I would add keys to each of these meta tags! Right now we have a lot of duplicate meta tags in the rendered head tag.

See [here](https://nextjs.org/docs/api-reference/next/head)

![Screen Shot 2021-08-17 at 12 08 59 PM](https://user-images.githubusercontent.com/44623317/129787521-38690903-3bcb-4ff6-85e0-fd8ec63863bf.png)

Reply:
Ooh good callout, I didn't really give too much thought into this one - I kind of assumed it behaved similar to an object spread where I could set defaults and overwrite them if needed!

Reply:
I believe the fix is to add a `key` property to the tags to allow them to be only generated once!

Or what I like to do is just define a `Layout` component that defines `<Head>` for me along with whatever `<Header>` and other things i need